src/platforms/javascript/common/performance/index.mdx

```diff
@@ -1,18 +1,32 @@
 ---
-title: Performance Monitoring
+title: Getting Started
 sidebar_order: 20
-description: "Learn how to enable performance monitoring in your app."
+description: "Learn how to enable performance monitoring in your app if it is not already set up."
 redirect_from:
   - /platforms/javascript/performance/apm-to-tracing/
 ---
 
-Sentry allows you to monitor the performance of your application, showing you how latency in one service may affect another service, and letting you pinpoint exactly which parts of a given operation may be responsible. To do this, it captures distributed traces consisting of transactions and spans, which measure individual services and individual operations within those services, respectively. You can learn more about this model in our [Distributed Tracing](/product/performance/distributed-tracing/) docs.
+<Note>
 
-Once tracing is enabled, certain types of operations will be measured automatically, and you can also choose to manually measure any operation you like. To learn more, see <PlatformLink to="/performance/capturing/automatic/">Capturing Transactions Automatically</PlatformLink> and <PlatformLink to="/performance/capturing/manual/">Capturing Transactions Manually</PlatformLink>.
+If you haven't already installed the package to enable performance monitoring when you set up your SDK, you'll need to install the package separately as documented on this page.
 
-## Enabling Tracing
+</Note>
 
-The first step is to install the tracing package, if you haven't done so already:
+Performance monitoring augments your existing error data by capturing interactions among your software systems.
+
+<Alert>
+
+Customers on any of our legacy plans will need to add transaction events to their subscription to access our performance monitoring features. For more details about access to these features, feel free to reach out at [performance-feedback@sentry.io](mailto:performance-feedback@sentry.io).
```

Comment:
I updated this text in #2729
```suggestion
Customers on legacy plans must add transaction events to their subscription in order to access our performance monitoring features. For general questions about performance monitoring with Sentry and for information about how you can add transactions to your plan, please contact [support@sentry.io](mailto:support@sentry.io). To start a performance monitoring trial and for information on volume pricing, please contact [sales@sentry.io](mailto:sales@sentry.io).
```

Reply:
I saw that after I'd pushed this PR to GH. I'll pull it in here and you'll close 2729?

Reply:
nvm - I see 2729 is merged. I'll fix any conflicts.